### PR TITLE
Add Debezium source/sink docs

### PR DESCRIPTION
**Pull request by mwylde**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (jbeisen) [inline .gitignore:1]: nit, but this is OS-specific so it kind of belongs in your global gitignore

File: connectors/postgres.mdx [hunk 2]
```diff
@@ -3,6 +3,211 @@ title: Postgres
...
-is currently in alpha. If you're interested in trying it out, reach out to the
-team on [Discord](https://discord.gg/cjCr5rVmyR) or at support@arroyo.systems.
+Arroyo supports Postgres as a source and sink, via [Debezium](https://debezium.io/). A
+native Postgres connector is planned for the future.
+
+Debezium is a CDC (change data capture) tool that monitors the Postgres transaction log
+and publishes changes to Kafka. Arroyo can read these changes from Kafka and present them
+as streaming tables which can be queried like any other table.
+
+Currently Debezium tables must be create via `CREATE TABLE` calls in SQL and must be configured
```
Comment (jbeisen) [inline connectors/postgres.mdx:13]:
```suggestion
Currently Debezium tables must be created via `CREATE TABLE` calls in SQL and must be configured
```

File: tutorial/debezium.mdx [hunk 3]
```diff
@@ -229,9 +229,8 @@ This will produce an updating dataset that reports how many running versus finis
 ### Writing Debezium outputs
 
 In addition to reading Debezium streams as update tables, Arroyo also supports
-writing Debezium messages to kafka as a sink. This can be done by creating a
-table with the `format` set to `debezium_json` and a `topic` set to
-the desired kafka topic.
+writing Debezium messages to kafka as a sink. See the
```
Comment (jbeisen) [inline tutorial/debezium.mdx:232]:
```suggestion
writing Debezium messages to Kafka as a sink. See the
```

**Review by jbeisen — APPROVED** [review]
Looks good, just some minor stuff

**Review by jacksonrnewhouse — APPROVED** [review]

**Pull request merged**